executors/readfile/README.md

```diff
@@ -0,0 +1,104 @@
+# Venom - Executor Read file
+
+Step for Read file
+
+Use case: you software write a file. Venom checks that file is produced, read it,
+and return content. Content can be used by another steps of testsuite.
+
+Path can contains a fullpath, a wildcard or a directory:
+
+```
+- path: /a/full/path/file.txt
+- path: afile.txt
+- path: *.yml
+- path: a_directory/
+```
```

Comment:
not this https://github.com/runabove/venom/pull/18/files#diff-a57396db6dc8d0930fa39fc20da70641R91 ?